packages/driver/cypress/integration/e2e/multi-domain/navigation_spec.ts

```diff
@@ -237,3 +237,31 @@ describe('delayed navigation', { experimentalSessionSupport: true, defaultComman
     })
   })
 })
+
+// @ts-ignore / session support is needed for visiting about:blank between tests
+describe('sub domain window loads before multi-domain command', { experimentalSessionSupport: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
+  })
+
+  it('establish a foobar spec bridge', () => {
+    return new Promise<void>((resolve) => {
+      setTimeout(() => {
+        cy.switchToDomain('http://foobar.com:3500', () => {
+          cy.get('a[data-cy="multi-domain-page"]')
+        }).then(resolve)
+      }, 1000)
+    })
+  })
+
+  it('page load before switchToDomain command when spec bridge is already established', () => {
+    return new Promise<void>((resolve) => {
+      setTimeout(() => {
```

Comment:
Since i cannot force the page to load faster, i'm delaying switchToDomain. Originally i had a test like this above, but when the spec bridge does not exist, we delay loading cross origin requests for 2000 ms to ensure the bridge is created before the request loads, so forcing a delay seemed invalid.

Reply:
Actually this got me curious in that we should delay for 2000 ms regardless of the state of the spec bridge.. With this PR i think i'm fixing a symptom, not the cause. It appears that the config tests are running so quickly we're getting messages mixed up, we send ready for domain and by the time we release the domain to load we've already moved on to the next test.

Reply:
The actual problem was elsewhere in another branch.